dictionary.go

```diff
@@ -96,13 +95,32 @@ func newDictionary(pkgs []*packages.Package, cfg config) (*dictionary, error) {
 			d.roots[p.Module.Dir] = true
 		}
 		for r := range d.roots {
-			err := d.AddDict(filepath.Join(r, ".words"))
+			err := ook.addDictionary(filepath.Join(r, ".words"))
 			if _, ok := err.(*os.PathError); !ok && err != nil {
 				return nil, err
 			}
 		}
 	}
 
+	// Load known words as a dictionary. This requires a write to
+	// disk since hunspell does not allow dictionaries to be loaded
+	// from memory, and affix rules can't be provided directly.
+	kw, err := os.CreateTemp("", "gospel")
+	if err != nil {
+		return nil, fmt.Errorf("failed to create known words dictionary: %v", err)
+	}
+	defer os.Remove(kw.Name())
```

Comment:
nit: you probably want to `defer kw.Close()` too, because some platforms like Windows will not let you remove open files. remember that os.File permits duplicate Close calls

Reply:
Ack